content/en/docs/reference/kubectl/conventions.md

```diff
@@ -37,23 +37,22 @@ All `kubectl run` generators are deprecated. See the Kubernetes v1.17 documentat
 
 #### Generators
 You can generate the following resources with a kubectl command, `kubectl create --dry-run=client -o yaml`:
-```
-  clusterrole         Create a ClusterRole.
-  clusterrolebinding  Create a ClusterRoleBinding for a particular ClusterRole.
-  configmap           Create a configmap from a local file, directory or literal value.
-  cronjob             Create a cronjob with the specified name.
-  deployment          Create a deployment with the specified name.
-  job                 Create a job with the specified name.
-  namespace           Create a namespace with the specified name.
-  poddisruptionbudget Create a pod disruption budget with the specified name.
-  priorityclass       Create a priorityclass with the specified name.
-  quota               Create a quota with the specified name.
-  role                Create a role with single rule.
-  rolebinding         Create a RoleBinding for a particular Role or ClusterRole.
-  secret              Create a secret using specified subcommand.
-  service             Create a service using specified subcommand.
-  serviceaccount      Create a service account with the specified name.
-```
+
+* `clusterrole`: Create a ClusterRole.
+* `clusterrolebinding`: Create a ClusterRoleBinding for a particular ClusterRole.
+* `configmap`: Create a configmap from a local file, directory or literal value.
+* `cronjob`: Create a cronjob with the specified name.
+* `deployment`: Create a deployment with the specified name.
+* `job`: Create a job with the specified name.
+* `namespace`: Create a namespace with the specified name.
+* `poddisruptionbudget`: Create a pod disruption budget with the specified name.
+* `priorityclass`: Create a priorityclass with the specified name.
+* `quota`: Create a quota with the specified name.
+* `role`: Create a role with single rule.
+* `rolebinding`: Create a RoleBinding for a particular Role or ClusterRole.
+* `secret`: Create a secret using specified subcommand.
+* `service`: Create a service using specified subcommand.
+* `serviceaccount`: Create a service account with the specified name.
```

Comment:
Please be consistent about resource type. For example, instead of "Create a configmap", we may want to put "Create a ConfigMap object" or "Create a `ConfigMap` object". Similarly, please revise "cronjob", "priorityclass" etc.

Reply:
Thanks, @tengqm - I have updated to CamelCase in https://github.com/kubernetes/website/pull/25549/commits/f095cf3ecf242cb2eee7bef9aa546ef2a8fdf126.